ssas/service/admin/api_test.go

```diff
@@ -161,6 +161,64 @@ func (s *APITestSuite) TestResetCredentials_InvalidSystemID() {
 	assert.Equal(s.T(), http.StatusNotFound, rr.Result().StatusCode)
 }
 
+func (s *APITestSuite) TestGetPublicKey() {
+	group := ssas.Group{GroupID: "api-test-get-public-key-group"}
+	err := s.db.Create(&group).Error
+	if err != nil {
+		s.FailNow(err.Error())
+	}
+
+	system := ssas.System{GroupID: group.GroupID, ClientID: "api-test-get-public-key-client"}
+	err = s.db.Create(&system).Error
+	if err != nil {
+		s.FailNow(err.Error())
+	}
+
+	key1Str := "publickey1"
+	encrKey1 := ssas.EncryptionKey{
+		SystemID: system.ID,
+		Body:     key1Str,
+	}
+	err = s.db.Create(&encrKey1).Error
+	if err != nil {
+		s.FailNow(err.Error())
+	}
+
+	key2Str := "publickey2"
```

Comment:
I might be missing why we need two public keys in this test.  There should only ever be one active per system.

Reply:
That was based on this criterion in the ticket: 
> If I submit two keys in a row (mimic-ing key rotation here), I get the second key I submitted, not the first

Reply:
Ah!  I knew I was missing something.  😄 
I read that requirement as, "double-check that our soft-deletion key rotation strategy works."  So that would involve calling `ssas.SavePublicKey()` instead of putting the keys directly in the database.